base/essentials.jl

```diff
@@ -17,6 +17,42 @@ end
 macro _noinline_meta()
     Expr(:meta, :noinline)
 end
+
+"""
+    @nospecialize
```

Comment:
Actually, not having it exported was pretty annoying. That's partly just an artifact of Base and tests using this more heavily than most code, but (1) other annotations like `@inline` are exported, and (2) it's hard to imagine `@nospecialize` conflicting with anything. So I'm more include to just export it.